app/api/v2/views.py

```diff
@@ -82,7 +82,7 @@ class CreateParcels(Resource):
 
 	def __init__(self):
 		"""Integrates RequestParser for input validation"""
-		self.user_id = 100
+		self.user_id = 0
```

Comment:
once you protect the routes with jwt token, you can pick the `user_id` from the token to avoid repetition.